modules/apps/jaggery/emm/src/emm/units/policy-create/public/js/policy-create.js

```diff
@@ -1948,11 +1969,11 @@ var showHideHelpText = function (addFormContainer) {
     }
 };
 
-function formatRepo (user) {
+function formatRepo(user) {
```

Comment:
what is the use of this function? function name is not descriptive enough to understand its usage